multicast/__main__.py

```
@@ -831,10 +831,10 @@ def cli() -> int:
 	__EXIT_CODE: tuple = (1, exceptions.EXIT_CODES[1][1])
 	if (sys.argv is not None) and (len(sys.argv) > 1):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)